tests/unit/src/test/scala/tests/ReferenceLspSuite.scala

```diff
@@ -212,4 +214,176 @@ class ReferenceLspSuite extends BaseLspSuite("reference") {
       _ = server.assertReferenceDefinitionBijection()
     } yield ()
   }
+
+  test("method-hierarchy") {
+    cleanWorkspace()
+    val expectedDiff =
```

Comment:
I was wrong, let's put it after the code definition, since it's more natural to read.